src/app/api/webhooks/route.ts

```diff
@@ -0,0 +1,74 @@
+// import Stripe from 'stripe';
+// import { buffer } from 'node:stream/consumers';
+// import micro_cors from 'micro-cors';
+
+// const stripe = new Stripe(process.env.STRIPE_SECRET_KEY!);
+
+// const webhookSecret: string = process.env.STRIPE_WEBHOOK_SECRET!;
+
+// // Stripe requires the raw body to construct the event.
```

Comment:
You don't want to leave heaps of "dead", commented out code kicking around - it will cause you issues later. Decide if you're going to keep it or remove it, but don't put it into master/main